src/components/SecondaryPanes/Breakpoints/index.js

```diff
@@ -81,41 +81,45 @@ class Breakpoints extends Component<Props> {
       ...breakpointSources.map(({ source, breakpoints }) => source)
     ];
 
-    return [
-      ...breakpointSources.map(({ source, breakpoints, i }) => {
-        const path = getDisplayPath(source, sources);
-        const sortedBreakpoints = sortSelectedBreakpoints(
-          breakpoints,
-          selectedSource
-        );
-
-        return [
-          <BreakpointHeading
-            source={source}
-            sources={sources}
-            path={path}
-            key={source.url}
-          />,
-          ...sortedBreakpoints.map(breakpoint => (
-            <Breakpoint
-              breakpoint={breakpoint}
-              source={source}
-              selectedSource={selectedSource}
-              key={makeBreakpointId(
-                getSelectedLocation(breakpoint, selectedSource)
-              )}
-            />
-          ))
-        ];
-      })
-    ];
+    return (
+      <div className="pane breakpoints-list">
+        {[
+          ...breakpointSources.map(({ source, breakpoints, i }) => {
```

Comment:
That was in the original code, but I'm pretty sure we don't need to do `{[ ...someArray.map(foo) ]}` to render a list in JSX, and can directly write `{someArray.map(foo)}`

Reply:
Good catch - thanks!